src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -16,34 +17,32 @@
 import seedu.address.model.person.Phone;
 import seedu.address.model.person.Points;
 import seedu.address.model.person.orders.Order;
-import seedu.address.model.tag.Tag;
 
 /**
  * Contains utility methods for populating {@code AddressBook} with sample data.
  */
 public class SampleDataUtil {
-    public static final MembershipPoints EMPTY_MEMSHIP = new MembershipPoints();
     public static Person[] getSamplePersons() {
         return new Person[] {
             new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
                 new Address("Blk 30 Geylang Street 29, #06-40"), new MembershipPoints(),
-                getTagSet("friends"), new Points("0"), getOrderArrayList("Cupcake x 3@2007-12-03T10:15:30")),
+                getAllergenSet("LF"), new Points("0"), getOrderArrayList("Cupcake x 3@2007-12-03T10:15:30")),
             new Person(new Name("Bernice Yu"), new Phone("99272758"), new Email("berniceyu@example.com"),
                 new Address("Blk 30 Lorong 3 Serangoon Gardens, #07-18"), new MembershipPoints(),
-                getTagSet("colleagues", "friends"), new Points("0"),
+                getAllergenSet("TN", "LF"), new Points("0"),
                     getOrderArrayList("Cookies x 5@2012-03-21T07:45:01", "Cupcake x 2@2014-12-11T13:01:02")),
             new Person(new Name("Charlotte Oliveiro"), new Phone("93210283"), new Email("charlotte@example.com"),
                 new Address("Blk 11 Ang Mo Kio Street 74, #11-04"), new MembershipPoints(),
-                getTagSet("neighbours"), new Points("0"), getOrderArrayList("Kaya Toast x 1@2024-01-01T07:00:00")),
+                getAllergenSet("EG"), new Points("0"), getOrderArrayList("Kaya Toast x 1@2024-01-01T07:00:00")),
             new Person(new Name("David Li"), new Phone("91031282"), new Email("lidavid@example.com"),
                 new Address("Blk 436 Serangoon Gardens Street 26, #16-43"), new MembershipPoints(),
-                getTagSet("family"), new Points("0"), getOrderArrayList()),
+                getAllergenSet("PN"), new Points("0"), getOrderArrayList()),
             new Person(new Name("Irfan Ibrahim"), new Phone("92492021"), new Email("irfan@example.com"),
                 new Address("Blk 47 Tampines Street 20, #17-35"), new MembershipPoints(),
-                getTagSet("classmates"), new Points("0"), getOrderArrayList("Cheese Tart x 6@2023-06-30T23:23:23")),
+                getAllergenSet("V"), new Points("0"), getOrderArrayList("Cheese Tart x 6@2023-06-30T23:23:23")),
             new Person(new Name("Roy Balakrishnan"), new Phone("92624417"), new Email("royb@example.com"),
                 new Address("Blk 45 Aljunied Street 85, #11-31"), new MembershipPoints(),
-                getTagSet("colleagues"), new Points("0"),
+                getAllergenSet("TN"), new Points("0"),
```

Comment:
Sorry, but what are "LF", "TN", "EG", "PN", and "V"? HAHA

Reply:
Lactose-free, Tree nuts, Egg, Peanuts, Vegan